Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -313,6 +315,14 @@ protected void onCreate(Bundle savedInstanceState)
 
     updateOrientation();
 
+    // Only android 9+ support this feature.
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P &&
+            BooleanSetting.MAIN_EXPAND_TO_CUTOUT_AREA.getBooleanGlobal())
```

Comment:
Since `mSettings` exists, I would recommend using it instead of using `getBooleanGlobal`. Not that it matters a lot, but I intended `getBooleanGlobal` as a kind of "escape hatch" for when you can't use the settings system in the intended way.